src/github.com/getlantern/flashlight/client/handler.go

```diff
@@ -49,7 +49,11 @@ func (client *Client) intercept(resp http.ResponseWriter, req *http.Request) {
 		respondBadGateway(resp, fmt.Sprintf("Unable to hijack connection: %s", err))
 		return
```

Comment:
This isn't related to your change @uaalto, but don't we need to close clientConn here?